_data/carousel-slider.yml

```diff
@@ -1,10 +1,10 @@
-- class: "item active"
+- class: "carousel-item active"
```

Comment:
I think it is bad to have CSS classes inside .yml files.
Do we really need the classes here?
If the first item needs the `active` class, we can use **liquid array methods**  to check if this is the first item in the array.

So I suggest remove all the `class` from this file, in order to keep it [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

Reply:
https://help.shopify.com/en/themes/liquid/objects/for-loops
    `{% if forloop.first == true %}`

Reply:
sure! i will fix this.